src/main.rs

```diff
@@ -84,7 +84,7 @@ fn main() -> Result<(), Box<dyn Error>> {
             return Err(err.into());
         }
     };
-    if matches.opt_present("help") || matches.free.len() < 2 {
```

Comment:
The input file became obsolete.

Reply:
I think input files should always be an option to the user. If they call the program without any parameters (and maybe a flag, so that the help will still be display by default), the program will parse `stdin`, and otherwise if a file (or maybe multiple, but that is another topic..) is supplied, it will just parse that.

Reply:
Suggestion for the behaviour:

1. If `--git` is specified, grab from the local Git history.
2. If `-i` is specified, read from the given file.
3. If none of the above is specified, read from `stdin`.

Reply:
Generally, yeah, I would like that. Just one thing:

If one does not know the program, and just runs it without any parameters, it will do nothing, since it is waiting for user input, and the user does not know, what they have to do. Maybe, that is fine and the user will think of using `--help` but idk.. 😅

Reply:
:wrench: :heavy_check_mark: